src/commands/createFunctionApp/stacks/FunctionAppStackStep.ts

```diff
@@ -20,6 +22,24 @@ export class FunctionAppStackStep extends AzureWizardPromptStep<IFunctionAppWiza
             if (!result) {
                 context.version = await promptForFuncVersion(context);
             } else {
+                const endOfLifeDate = result.minorVersion.stackSettings.linuxRuntimeSettings?.endOfLifeDate;
+                const settingKey: string = 'endOfLifeWarning';
+                if (getWorkspaceSetting<boolean>(settingKey)) {
+                    if (endOfLifeDate) {
+                        if (shouldShowEolWarning(endOfLifeDate)) {
+                            const message = localize('endOfLife', "The chosen runtime stack has an end of support deadline coming up. " +
+                                "After the deadline, function apps can be created and deployed, and existing apps continue to run. " +
+                                "However, your apps won't be eligible for new features, security patches, performance optimizations, and support until you upgrade them");
```

Comment:
nit: I like using backticks for multiline strings

```suggestion
                            const message = localize('endOfLife', `The chosen runtime stack has an end of support deadline coming up. 
                                After the deadline, function apps can be created and deployed, and existing apps continue to run. 
                                However, your apps won't be eligible for new features, security patches, performance optimizations, and support until you upgrade them`);
```

Reply:
I added this suggestion and it seems to be doing something weird with the warning message.
![image](https://user-images.githubusercontent.com/59709511/200078187-22dd2e15-1141-4f0d-9c6c-1c93202a646f.png)

Reply:
😱

My bad, of course the newlines will show up when using backticks. Change it back to how it was, or use the `\` character. 

example from docs:
```js
const longString = "This is a very long string which needs \
to wrap across multiple lines because \
otherwise my code is unreadable."
```

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String#long_literal_strings

Reply:
OK! With the `\` you can't include indents which I don't think looks too good so I just changed is back to how it was!